ee/codegen/src/generators/nodes/code-execution-node.ts

```diff
@@ -112,6 +113,16 @@ export class CodeExecutionNode extends BaseSingleFileNode<
       })
     );
 
+    const argId = this.nodeInputsByKey.get("arg")?.nodeInputData.id;
+    if (argId) {
+      statements.push(
+        python.field({
+          name: "arg_input_id",
+          initializer: python.TypeInstantiation.uuid(argId),
+        })
+      );
+    }
```

Comment:
[blocking] Code execution nodes have dynamic inputs for their arguments. Meaning, it's guaranteed for all code execution nodes to have a node input called `arg_input_id`. It just happens that this fixture has a single node input with this name.

Code execution nodes behave more similarly to inline prompt nodes or templating nodes, where they have a class attribute call `code_inputs` that defines a mapping of their inputs. We need to generate that. I suggest looking at templating node's `inputs` class attribute for the pattern we should use.

Reply:
Updated